MongoDB/node.count.js

```diff
@@ -0,0 +1,25 @@
+const _ = require('lodash');
+let logger = global.logger;
+
+module.exports = async (connectorData, inputData) => {
+    logger.trace("MongoDB Count Node: Invoked!");
+
+    try {
+        logger.trace(`MongoDB Count Node: Input - ${JSON.stringify(inputData)}`);
+
+        if (!connectorData.db || !inputData.collection) {
```

Comment:
Assume `connector.db` to be always there. Collection name is a mandatory field. 
Add check for collection name in all nodes, please